pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/SimpleBrokerStartTest.java

```diff
@@ -97,4 +101,31 @@ public void testNoNICSpeed() throws Exception {
     }
 
 
+    @Test
+    public void testCGroupMetrics() throws IllegalAccessException {
+        if (!LinuxInfoUtils.isLinux()) {
+            return;
+        }
+
+        boolean existsCGroup = Files.exists(Paths.get("/sys/fs/cgroup"));
+        boolean cGroupEnabled = LinuxInfoUtils.isCGroupEnabled();
+        Assert.assertEquals(cGroupEnabled, existsCGroup);
+
+        double totalCpuLimit = LinuxInfoUtils.getTotalCpuLimit(cGroupEnabled);
+        log.info("totalCpuLimit: {}", totalCpuLimit);
+        Assert.assertTrue(totalCpuLimit > 0.0);
+
+        if (cGroupEnabled) {
+            Assert.assertNotNull(FieldUtils.readStaticField(LinuxInfoUtils.class, "metrics", true));
```

Comment:
I think you can add a getter method with `@VisibleForTesting` for these fields to avoid reflect.

Reply:
Why not use reflection to get it? This is just a test.

Reply:
This test may be broken when someone modifies these fields.

Reply:
Ok, it makes sense.